tests/ports/psoc6/run_psoc6_tests.sh

```diff
@@ -341,6 +347,17 @@ run_ci_tests() {
       fi
     fi
     i2s_tests
+
+    if [ "${board}" == "CY8CPROTO-062-4343W" ] || [ "${board}" == "CY8CPROTO-063-BLE" ]; then
+      dev_test=${devs_b[0]}
```

Comment:
https://github.com/Infineon/micropython/pull/195/files#diff-cafe99212312cbfc6caa307a421984d9049bb0145883517444e505039fc46aebR287-R307

Here we have to update the version to 0.6.0 (only for CY8CPROTO-062-4343W) to match the diagram version.
And in the {hil-name}-devs.yml file we have to also update that version.

Reply:
Done :)